Backend/Backend/Models/User.cs

```diff
@@ -39,5 +39,39 @@ public User()
     [Required]
     public string Password { get; set; }
 
+    public string Username { get; set; }
+
+    /// <summary>
+    /// URL for user's PFP
+    /// </summary>
+    public string Avatar { get; set; }
+
+    public string[] Interests { get; set; }
     public DateTime DateOfBirth { get; set; }
+    
+    [DefaultValue(GenderEnum.Other)]
+    public GenderEnum Gender { get; set; }
+
+    /// <summary>
+    /// Flag indicating if the user is a podcaster
+    /// </summary>
+    public bool IsPodcaster { get; set; }
+
+    public ICollection<Podcast> Podcasts { get; } = new List<Podcast>();
```

Comment:
All the Icollections should be nullables, only if the person isPodcaster are most of these on anyways

Reply:
Nullable = Empty collection No?

Reply:
Yeah you are right. Wasnt thinking